### PR TITLE
Set Test-runner to false for alphametics

### PR DESCRIPTION
**Pull request by BethanyG**

### CODE REVIEW

File: bin/data.py [hunk 1]
```diff
@@ -134,6 +134,7 @@ class ExerciseConfig:
     forked_from: str = None
     contributors: List[str] = None
     language_versions: List[str] = None
+    test_runner: bool = True
```
Comment (BethanyG) [inline bin/data.py:137]:
```suggestion
```

File: exercises/practice/alphametics/.meta/config.json [hunk 2]
```diff
@@ -26,5 +26,6 @@
     "example": [
       ".meta/example.py"
     ]
-  }
+  },
+  "test_runner": "false"
```
Comment (iHiD) [inline exercises/practice/alphametics/.meta/config.json:30]: I **think** this needs to be a boolean, not a string.

Comment (BethanyG) [reply exercises/practice/alphametics/.meta/config.json:30]:
```suggestion
  "test_runner": false
```

**Comment by iHiD** [comment]
Thanks :)

**Pull request merged**